src/main/java/frc/robot/Constants.java

```diff
@@ -10,45 +10,38 @@
 import edu.wpi.first.math.geometry.Transform3d;
 import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
 
 
 public class Constants {
-    public static final class DriveConstants {
-        public static final int Left_Front_ID = 1;
-        public static final int Right_Front_ID = 3;
-        public static final int Left_Back_ID = 2;
-        public static final int Right_Back_ID = 4;
 
+    public static final class GlobalConstants{
```

Comment:
Why global constants? A lot of the ones that are taken out of DriveConstants are *specific to the drivetrain*. The only ones that might fit here are falcon counts per revolution (and im not convinced that *needs* a constant) and the PDH object (and again not sure thats needed).

Reply:
good point (honestly I forgot I even made GlobalConstants) . Just delete my changes and move the pdh object to the ArmConstants Class